task/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+    content = models.TextField()
+    created = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField(null=True, blank=True)
+    task_done = models.BooleanField()
+    tags = models.ManyToManyField(Tag, related_name="tasks", blank=True)
+
+    class Meta:
+        ordering = ("created",)
```

Comment:
According to the task requirements, the ordering must be from "not done" to "done" and from newest to oldest.

Reply:
class TaskListView(generic.ListView):
    model = Task
    paginate_by = 4
    template_name = "task/index.html"

    def get_queryset(self):
        queryset = Task.objects.order_by("task_done")
        return queryset